api/views.py

```diff
@@ -93,4 +92,10 @@ def change_seat_status(request, pk):
     serialseats = ChangeReservationSeatStatusSerializer(seats, many=False)
     if not seats.is_occupied:
         seats.is_occupied = True
-    return Response(serialseats.data)
\ No newline at end of file
+    return Response(serialseats.data)
+
+
+class PossibleFraudsReservations(viewsets.ReadOnlyModelViewSet):
+    queryset = Reservation.objects.filter(seat__is_occupied=True, is_confirmed=False)
```

Comment:
Maybe it's just someone cleaning the hall that sat on the seat. I think it would be useful to also check if the movie for this reservation has started playing.